### ISSUE TITLE
Blur effect greys out the image completely

### ISSUE THREAD

**Issue by jamesfzhang** [react-native-fellowship/react-native-blur#37]
Hello, I'm trying to add a blur effect but it seems off. Here's the image with and without this library. I tried both `BlurView` and `VibrantView`:

<img width="370" alt="screen shot 2015-12-15 at 3 52 29 pm" src="https://cloud.githubusercontent.com/assets/2057670/11827845/1e5d79e6-a344-11e5-8277-fbc06552ca8e.png">
<img width="373" alt="screen shot 2015-12-15 at 3 53 03 pm" src="https://cloud.githubusercontent.com/assets/2057670/11827846/1e5edb24-a344-11e5-9f63-177c53859ad8.png">

Here's the code. Any ideas?

```js
<View style={styles.contentContainer}>
  <Image
    style={styles.image}
    source={this.getImage()}>
    <BlurView blurType='light' style={styles.image}/>
  </Image>
</View>
```

**Comment by ethandt92** [comment]
ran into same issue. blur view needs ```background: 'transparent'``` to work like in the example.

**Comment by Kureev** [comment]
Solved in `v0.7.7`
